packages/runtime/src/templates/edge/runtime.ts

```diff
@@ -18,8 +18,9 @@ export interface RequestData {
     city?: string
     country?: string
     region?: string
-    latitude?: string
-    longitude?: string
+    latitude?: number
```

Comment:
Just updating this as our context object type has it marked as number for longitude and latitude.